docs/pretrained.rst

```diff
@@ -185,6 +185,15 @@ The input output configuration is as follows:
 
     - hovernet_fast-pannuke
 
+.. collapse:: Output Nuclear Classes
+
+    - 0: Background
```

Comment:
Does this class exist in the TIAToolbox version?

Reply:
No- changing